### ISSUE TITLE
model select and sample strategy

### ISSUE THREAD

**Issue by Lycus99** [ttlmh/Bridge-Prompt#9]
Thanks for your interesting work!
When I repeat your work on my dataset with the task of action segmentation, I have another two questions:
1. how to select the best backbone model? I try to use the loss on the validation set to select, but the embeddings extracted by the minimal validation loss model are not working well when I use Asformer as the temporal classifier. 
2. have you ever compared a different strategy of downsampling and overlap? I wonder if the model works well on the action clip that contains 4-5 actions than contains 2-3 actions.

**Comment by ttlmh** [comment]
Hi, thanks for your interest in our work!

Our experiments simply took the trained backbone model from the last epoch as the final feature extractor. Thus, the problem may lie in choosing the appropriate sampling strategy. I think one possible solution is to combine different downsampling rates and overlaps. During our sampling, we included both the window with 2-3 actions or 4-5 actions in the training set. Another problem may lie in the form of labeling. We believe that more detailed annotations facilitate the learning of visual-language models. For instance, as for the GTEA dataset, we took the raw labels (74 classes including both verbs and nouns) instead of the predicted labels for action segmentation (11 classes including only verbs).

Thanks!
